UI/Web/src/theme/components/_sidenav.scss

```diff
@@ -1,171 +1,236 @@
 @import "../variables";
 
 .side-nav-container {
-    padding-bottom: 10px;
-    width: 190px;
-    background-color: var(--side-nav-bg-color);
-    height: calc((var(--vh) * 100) - 115px);
-    position: fixed;
-    margin: 0;
-    left: 10px;
-    top: 73px;
-    border-radius: var(--side-nav-border-radius);
-    transition:
-        width var(--side-nav-openclose-transition),
-        background-color var(--side-nav-bg-color-transition),
-        border-color var(--side-nav-border-transition);
-    border: var(--side-nav-border);
+  width: var(--side-nav-width);
+  background-color: var(--side-nav-bg-color);
+  height: calc((var(--vh) * 100) - 115px);
+  position: fixed;
+  margin: 0;
+  left: 10px;
+  top: 64px;
+  border-radius: var(--side-nav-border-radius);
+  transition: width var(--side-nav-openclose-transition), background-color var(--side-nav-bg-color-transition), border-color var(--side-nav-border-transition);
+  border: var(--side-nav-border);
+
+  &::-webkit-scrollbar {
+    visibility: hidden;
+  }
+
+  &.preference {
+    height: calc((var(--vh)*100) - 75px);
+    width: 238px;
 
-    &::-webkit-scrollbar {
-        visibility: hidden;
-    }
+    .side-nav {
+      overflow-x: hidden;
 
-    &.preference {
-        height: calc((var(--vh) * 100));
-    }
+      .side-nav-header {
+        color: rgba(255, 255, 255, 0.3);
+        font-size: 15px;
+        &:first-of-type {
+            margin-top: 0.7rem;
+        }
+      }
+
+      .side-nav-item {
+        font-size: 0.9rem;
+        min-height: 30px;
+        padding-left: 12px;
+        justify-content: unset;
+
+        &.active {
+          .active-highlight {
+            width: 4px;
+            height: 4px;
+            border-radius: 50px;
+            opacity: 1;
+            position: unset;
+          }
+        }
 
-    &.no-donate {
-        height: calc((var(--vh) * 100) - 82px);
-    }
+        .active-highlight {
+          width: 4px;
+          height: 4px;
+          border-radius: 50px;
+          opacity: 0;
+          margin-left: 13px;
+        }
 
-    &.hidden {
-        display: none;
-        opacity: 0;
-    }
+        .side-nav-text {
+          text-align: unset;
+          width: 100%;
+          margin-left: 7px;
+          flex-grow: 1;
 
-    &.closed {
-        width: 55px;
-        overflow-x: hidden;
-        overflow-y: hidden;
-        background-color: var(--side-nav-closed-bg-color);
-        border: var(--side-nav-border-closed);
-    }
+          div {
+            min-width: unset;
+            width: unset;
+            text-overflow: ellipsis;
+            display: block;
+            font-size: 13px;
```

Comment:
Far too small and creates a ux discrepency between the home page sidebar and the preference sidebar. Let's adjust this to a manageable size for mobile as well.

Reply:
I’ll increase the font size for mobile, I took inspiration from the Plex settings side bar and thought that Kavita’s UX seems inspired by Plex so I tried to match that. I can increase the font size if needed along with mobile.